.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: go
 
 go:
-            - 1.7
+            - 1.9
```

Comment:
You can specify multiple targets. I'd suggest including 1.10 and 1.11 as well.

Reply:
NB: 1.10 needs to be specified as a string.
https://docs.travis-ci.com/user/languages/go/#specifying-a-go-version-to-use

Reply:
Done